src/importsource.cpp

```diff
@@ -14,12 +14,20 @@ See the License for the specific language governing permissions and
 limitations under the License.
 */
 
-#include "libcellml/importsource.h"
+#include <algorithm> // Adding because Ubuntu build complained?
+#include <vector>
 
+#include "libcellml/component.h"
+#include "libcellml/importedentity.h"
+#include "libcellml/importsource.h"
```

Comment:
Remove this include (since it should be included at the top).

Reply:
81328612